x-pack/plugins/siem/public/management/store/middleware.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ImmutableMiddlewareFactory, substateMiddlewareFactory } from '../../common/store';
+import { ManagementState } from '../types';
+import { policyListMiddlewareFactory } from '../pages/policy/store/policy_list';
+import { policyDetailsMiddlewareFactory } from '../pages/policy/store/policy_details';
+import {
+  MANAGEMENT_STORE_GLOBAL_NAMESPACE,
+  MANAGEMENT_STORE_POLICY_DETAILS_NAMESPACE,
+  MANAGEMENT_STORE_POLICY_LIST_NAMESPACE,
+} from '../common/constants';
+
+export const managementMiddlewareFactory = (coreStart, depsStart) => {
```

Comment:
this is the reason I changed the `middleware` prop for sub-plugins to be an array.